### ISSUE TITLE
[🐛 | Bug]: last edition time difference

### ISSUE THREAD

**Issue by Landeers** [Scille/parsec-cloud#5693]
### Parsec version tested on:

2.15.0

### Platforms tested on:

Windows

### Bug description:

Time displayed into file explorer is not good, the time moves forward by one hour in the Paris time zone (winter hour).

### Relevant output:

![image](https://github.com/Scille/parsec-cloud/assets/10208225/2572d416-2beb-4f2f-b91a-ff23bf47bc89)

**Comment by vxgmichel** [comment]
It's a regression dating from this change: https://github.com/Scille/parsec-cloud/commit/cf9cd42d37215c917899b84647df5c8c2dd7e68d#diff-f100065badef0b87ebff34dcfedef21ec866f6e6d89d7aff7ec195326f50cadeL162-R163

It's been introduced in version 2.12 and has not been detected ever since.

The issue here is that `datetime.fromtimestamp` returns a naive local datetime while `pendulum.from_timestamp` returns an UTC datetime, so the following diff should be enough to fix the issue:
```diff
diff --git a/parsec/core/mountpoint/winfsp_operations.py b/parsec/core/mountpoint/winfsp_operations.py
index 71099526a..ecf0bd6b7 100644
--- a/parsec/core/mountpoint/winfsp_operations.py
+++ b/parsec/core/mountpoint/winfsp_operations.py
@@ -3,7 +3,7 @@ from __future__ import annotations
 
 import json
 from contextlib import contextmanager
-from datetime import datetime
+from datetime import datetime, timezone
 from functools import partial, wraps
 from pathlib import PurePath
 from typing import Any, Callable, Iterator, List, Tuple, TypeVar, Union, cast
@@ -220,8 +220,8 @@ def stat_to_file_attributes(stat: dict[str, str]) -> FILE_ATTRIBUTE:
 
 
 def stat_to_winfsp_attributes(stat: dict[str, Any]) -> dict[str, Any]:
-    created = dt_to_filetime(datetime.fromtimestamp(stat["created"].timestamp()))
-    updated = dt_to_filetime(datetime.fromtimestamp(stat["updated"].timestamp()))
+    created = dt_to_filetime(datetime.fromtimestamp(stat["created"].timestamp(), tz=timezone.UTC))
+    updated = dt_to_filetime(datetime.fromtimestamp(stat["updated"].timestamp(), tz=timezone.UTC))
     attributes = {
         "creation_time": created,
         "last_access_time": updated,
```

**Comment by FirelightFlagboy** [comment]
Fixed in #5752
